### DISCUSSION TITLE
How to import NPM package to the global namespace client side only?

### DISCUSSION THREAD

**Discussion by SeanGroff** [vercel/next.js#31967] [Help]
I'm trying to use the NPM package [amazon-connect-streams](https://github.com/amazon-connect/amazon-connect-streams#importing-streams-with-npm-and-es6)

When running `npm run dev` I receive a build failure `error - ReferenceError: navigator is not defined`

The code that causes this error is the `import` of `amazon-connect-streams`

`import 'amazon-connect-streams'`

This import adds the package to the global namespace and uses the `navigator` browser API.

**_Since this is a client only library, I only want to import this npm package on the client._**

I tried:

```
import dynamic from 'next/dynamic';

dynamic(() => import('amazon-connect-streams'), { ssr: false });
```

This didn't work and fails to add `connect` to the `window` object.

**Comment by vpaflah** [comment]
did you find any solution?
